uiowa/uiowa
# Added json file extension

Closes #3784 
# How to test


1. `git checkout json-media`
2. `blt ds --site=sandbox`
3. to be on safe side - can `drush @sandbox.local cim` & `drush @sandbox.local cr`
3. `drush @sandbox.local uli`
4. Go to https://sandbox.local.drupal.uiowa.edu/media/add/file and add a json file
5. make sure can link to existing json file you created
6. Also add a new json file to WYSWYG to make sure adding a new json file in Media Library works correctly as well
